terraform/api/aws/versions.tf

```diff
@@ -6,7 +6,7 @@ terraform {
     }
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = "~> 1.11"
+      version = "~> 2.9.0"
```

Comment:
@heronrs is there any breaking change between these versions?

Reply:
yes, not many though. I included the changes in the commit.

https://github.com/convox/convox/pull/430/files#diff-e192f28400f22cc7c340e60537f67c09e00764c724f746226bd2d5b323aa696bR110
https://github.com/convox/convox/pull/430/files#diff-42e7726c119a8a8da585260c23ea0e7d38014259b9a791400a4b73b86448574fR61